### ISSUE TITLE
Feature Request: Exclude list for auto-reload

### ISSUE THREAD

**Issue by joshhudnall** [bdkjones/CodeKit-1-Old#674]
When developing, I'm often jumping back and forth between multiple tabs and Coda to make changes. This means that reloading the active tab is not useful to me because I could be looking at documentation when I make a change. 

However, just being able to specify strings to look for to reload isn't quite enough either. When developing locally, all of my sites are served under the url [domain].dev, so I set it to only reload sites with the string ".dev". This works, but it also reloads my local phpmyadmin.dev page and the wp-admin page if I'm working on a Wordpress site. It would be great to have a deny list as well.

It's not practical for me to jump back and forth between Safari and Chrome to avoid this. I have a super fast transition Command-tabbing back and forth, but adding a third app into the mix means having to figure out where I am and where I want to go next, each time shaving valuable seconds off of every transition. I think that's an important consideration for any strong web dev.

Thanks.
Josh

**Comment by joshhudnall** [comment]
Closing because it will clearly never be addressed and I've found a workaround.

**Comment by bdkjones** [comment]
Hey Josh,

You should _definitely_ not be on CodeKit 1 any longer. It was discontinued over a year ago and all the compilers in it are about 1.5 years out of date. They'll fail to compile modern syntax. 

I know 2.0 is an $18 upgrade, but it's WELL worth it and your money lets me keep improving the app. The 2.0 browser-refreshing workflow will solve this issue as well.

**Comment by joshhudnall** [comment]
@bdkjones I posted the issue back in 2013 before there was a 2.0. I upgraded to 2.0 on day one—the price was no issue for such a useful app. :) But I went a different route for browser refreshing anyway.

Thanks for the reply. I mostly closed it so it didn't show up in my open issues list anymore. I guess my comments were just in response to the fact that I never got any kind of reply. No big deal now though.

Cheers.
Josh
